schemas/rum/view-schema.json

```diff
@@ -313,6 +313,30 @@
               "description": "Version of the update of the view event",
               "minimum": 0,
               "readOnly": true
+            },
+            "page_states": {
+              "type": "array",
+              "description": "List of the page states during the view",
+              "items": {
+                "type": "object",
+                "description": "Properties of the page state",
+                "required": ["state", "start"],
```

Comment:
For the `in_foreground_periods` attributes, we also keep a `duration` attribute, would it make sense to add it to the `page_states`?

Reply:
For browser SDK investigation it's not needed. Plus, we can still calculate the duration like so: 
`page_states[n + 1].start - page_states[n].start `